parsec/core/remote_devices_manager.py

```diff
@@ -102,9 +102,7 @@ async def get_user(
             )
         return verified_user, verified_revoked_user
 
-    async def get_device(
-        self, device_id: Optional[DeviceID], no_cache: bool = False
-    ) -> DeviceCertificate:
+    async def get_device(self, device_id: DeviceID, no_cache: bool = False) -> DeviceCertificate:
```

Comment:
Why is `device_id` no longer optional? Are we sure it never happens?

Reply:
This getter calls `trustchain_ctx.get_device` which needs a `DeviceID` parameter. If this parameter is `None` this will raise an Exception because `None` can't be converted to a DeviceID object.